filecoin-proofs/src/api/post.rs

```diff
@@ -151,6 +151,8 @@ pub fn generate_candidates(
     replicas: &BTreeMap<SectorId, PrivateReplicaInfo>,
     prover_id: ProverId,
 ) -> Result<Vec<Candidate>> {
+    info!("generate_candidates:start");
```

Comment:
FWIW: All the filecoin-ffi bindings put a space between `thing: start` and `thing: end`.